src/openvic-simulation/military/Unit.cpp

```diff
@@ -204,6 +204,8 @@ bool UnitManager::generate_modifiers(ModifierManager& modifier_manager) const {
 
 		ret &= modifier_manager.register_complex_modifier(identifier);
 
+		stat_modifier("_attack", true, RAW_DECIMAL);
```

Comment:
Do ships also use attack and defence modifiers?
I couldn't find an example of that in vanilla.

Reply:
I didn't think they did, but i found it in some mods and when I tested it did show up in the invention tooltips. Also when you check the tooltip of a naval unit in the military menu, it lists their attack and defence stats. We should check if the modifier actually has an effect, but even if it doesn't we'll still want it registered as a valid key, just with a warning that it doesn't have an effect.